thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/AutometricsConfigBean.java

```diff
@@ -0,0 +1,121 @@
+package com.linkedin.thirdeye.datalayer.pojo;
+
+import java.util.Objects;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+
+/**
+ * Class to represent the autometric definition in thirdeye database
+ */
+public class AutometricsConfigBean extends AbstractBean {
+
+  /** Metric name as in MetricConfig **/
+  private String metric;
+  /** Dashboard name in external dashboard, and dataset name in datasetConfig **/
+  private String dashboard;
+  private String rrd;
+  private String metricDataType;
+  /**  COUNTER or GAUGE **/
+  private String autometricsType;
```

Comment:
I did not add enum here, because that would mean putting autometrics specific logic in open source